### PR TITLE
added info on configuring branches to be displayed and reordered few …

### PR DESCRIPTION
**Pull request by abhijitkini**
…sections based on the UI

### CODE REVIEW

**Comment by abhijitkini** [comment]
https://github.com/Shippable/docsv2/issues/324

File: sources/ci_subscriptions.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ The `CI` tab of your Subscription page shows you the latest status of all enable
 
 <img src="../images/subscription_dashboard.png" alt="Subscription Dashboard" style="width:800px;"/>
 
-The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all enabled projects. Latest status is defined as the build status for the latest commit build on the default branch. If no default branch exists for your project, project status is latest commit build status for the master branch, and if no master branch exists, then the project status is the build status of latest commit build across all branches.
+The **Summary View** is the default when you navigate to this dashboard. It shows you the latest status for all enabled projects. Latest status is defined as the build status for the latest commit run or pull request run across all configured branches. More details on this in the [Dashboard Settings for Projects](ci_projects/#dashboard-settings). Your default branch and 'Show only Commit runs' are set as default in the Summary view.
```
Comment (manishas) [inline sources/ci_subscriptions.md]: By default, the latest status for a project is the build status for the latest commit for the default branch configured in your source control account. You can customize the branch(es) shown on this page and also configure this view to include status for pull request builds. More details on this in the [Dashboard Settings for Projects](ci_projects/#dashboard-settings).

File: sources/ci_subscriptions.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The `CI` tab of your Subscription page shows you the latest status of all enable
...
 
 If a build is queued or in progress, you will see at the top of the summary view.
```
Comment (manishas) [inline sources/ci_subscriptions.md]: If a build is queued or in progress, you will see it at the top of the Summary view.

File: sources/ci_projects.md [hunk 3]
```diff
@@ -34,21 +38,27 @@ To rebuild a previous build, go to the **History view** and click on `Rebuild` f
...
 ## Settings
-You can perform project level actions by clicking on the `Settings` tab on the Project page.
+You can perform project level actions by clicking on the `Settings` tab on the Project page as shown below:
+
+<img src="../images/Project_settings.png" alt="Project Settings" style="width:700px;"/>
+
+
+### Pausing your project
+You can 'Pause' a project that will stop all web hooks triggering for this project. This means Shippable will stop listening to all code commits and pull requests on your source control repository and no action will be taken.
+
```
Comment (manishas) [inline sources/ci_projects.md]: You can 'Pause' a project to stop triggering builds for that project.

File: sources/ci_projects.md [hunk 4]
```diff
@@ -95,6 +102,19 @@ Select the branch you want the status for and pick between the 'Image URL' or 'M
 You can use this code to display your code coverage report after code coverage runs. The badge displays appropriate colors depending on the code coverage configuration in the `shippable.yml`. Refer the  [documentation](http://docs.shippable.com/ci_configure/#test_code_coverage) on how to configure code coverage reports.
 
 
+### Dashboard Settings
+You can configure the branches and run types from which Shippable shows latest runs in the 'Status' section of the 'Summary' tab of the Subscription & Home pages.
+
```
Comment (manishas) [inline sources/ci_projects.md]: By default, we show a project's status on the Subscription and Landing pages with the following logic: show status of latest commit build of default branch configured in source control repo; if no default branch, show latest commit build for master branch; if no master branch, show status of the latest build across branches. Builds triggered for pull requests are not considered while determining status of a branch.

You can override this default behavior in the `Dashboard Settings` section by 
- choosing the branch(es) that you want displayed on the Subscription and Landing pages, and,
- specifying that you want builds triggered by pull requests to be considered while computing status for a project.

For example, the configuration below will show status of the latest build for branches `master` and `revert-68-master`, irrespective of whether it was triggered by a commit or a pull request.

File: sources/ci_projects.md [hunk 5]
```diff
@@ -95,6 +102,19 @@ Select the branch you want the status for and pick between the 'Image URL' or 'M
...
+**Branches to be displayed**: Configure the branches for which runs should be displayed in the 'Status' section of the 'Summary' tab of Subscription & Home pages. By default, the default branch of your repository/project is displayed.
+
+**Run types to be displayed**: Configure the type of Runs to be displayed in the 'Status' section of the 'Summary' tab of Subscription & Home pages. You can opt to show only commit runs or both commit and pull request runs. The default setting is set to 'Commits Only'.
+
+<img src="../images/project_settings_dashboard.png" alt="Project Settings Dashboard configuration" style="width:700px;"/>
+
```
Comment (manishas) [inline sources/ci_projects.md]: delete the above.

**Comment by manishas** [comment]
@abhijitkini pls make the changes above. i have also sent you the picture we should use for the branches config.

**Pull request merged**